mast_aladin_lite/aida.py

```diff
@@ -34,12 +37,38 @@ def set_viewport(self, center):
 
         """
 
-        if not isinstance(center, SkyCoord):
-            raise TypeError(
-                "`center` must be a SkyCoord object."
-            )
+        if center is not None:
+            if not isinstance(center, SkyCoord):
+                raise TypeError(
+                    "`center` must be a SkyCoord object."
+                )
+
+            self.app.target = center
+
+        if fov is not None:
+            scale_factor = 0
+            if isinstance(fov, (u.Quantity, Angle)):
+                fov = fov.value
+
+            elif isinstance(fov, (float, int)):
+                if fov < 1:
+                    scale_factor = fov
 
-        self.app.target = center
+            else:
+                raise ValueError(
+                    f"`fov` must be an `~astropy.coordinates.Angle` or float, got {fov=}"
+                )
+
+            current_fov = self.app.fov.value
```

Comment:
To make this safer and clearer:
```suggestion
            current_fov = self.app.fov.to_value(u.deg)
```